Cargo.toml

```diff
@@ -1,29 +1,29 @@
 [package]
 name = "quilt-installer"
 description = "The installer for quilt-loader"
-
 version = "0.1.1"
 license = "Apache-2.0"
-
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-anyhow = "1.0"
-async-trait = "0.1"
-base64 = "0.13"
-chrono = "0.4"
-clap = { version = "3.2", features = ["derive"] }
-iced = { version = "0.3", default-features = false, features = ["glow", "tokio"] }
-iced_glow = "0.2"
-native-dialog = "0.6.3"
-png = "0.17.5"
 reqwest = { version = "0.11", features = ["blocking", "json"] }
+iced = { version = "0.6", default-features = false, features = [
+    "glow",
+    "tokio",
+] }
```

Comment:
hm, I feel like splitting this into multiple lines might be bad here

Reply:
That was just my toml formatter. I think this line was too long , but I'll revert it if you prefer.